### ISSUE TITLE
Filter options explore page/insitutional page 

### ISSUE THREAD

**Issue by Grizzly127** [Kompakkt/Repo#54]
### Annotation options

We thought about filtering the objects after annotation options: 
<img width="301" alt="Bildschirmfoto 2024-12-05 um 10 54 35" src="https://github.com/user-attachments/assets/4a196f97-aa7f-4895-b7e7-b25836d2c225">

We got the feedback from users that it is a bit confusing and maybe not even a use case. 
Users can always annotate objects in their collections. 
The information if it is already annotated is not something that the user would look for, it is more an information that is displayed looking at the object. Therefore I suggest removing this as a filter option. 

What is sometimes not clear to the user is that objects can only be annotated if they are in a collection, therefore we should address this more clearly.
We could directly show the users collections here. The button is blue, because the user is able to annotate, but only needs to add the object to a collection. 
<img width="555" alt="Bildschirmfoto 2024-12-05 um 11 24 04" src="https://github.com/user-attachments/assets/d45d5e95-9e5a-47ad-add5-0b8fbd9d9da1">

Right now the tooltip for objects that are not annotable only says: 
<img width="470" alt="Bildschirmfoto 2024-12-05 um 11 04 41" src="https://github.com/user-attachments/assets/4916ed7a-07ab-4ffb-8df4-e71e3096fe67">

We should make it more clear why the user is not able to annotate and maybe add (or something else, I am not sure how it is organized right now): 
<img width="374" alt="Bildschirmfoto 2024-12-05 um 11 35 26" src="https://github.com/user-attachments/assets/8df38b99-3ac6-4cd1-a6a1-8c90adf434db">

@lozanaross @ZetOE @HeyItsBATMAN

**Comment by HeyItsBATMAN** [comment]
I strongly prefer the route of users always being able to press the annotate button.

When they press it, they get a list of collections.
The list should display some information on the collections such as:
- whether the object already exists in the collection
- if the object already is in the collection, how many annotations it already has
- the owner of the collection (because you could also possibly want to annotate in someone elses collection, in which you have the permission to do so)

If this does not fit into a list, it might make sense to create this as a modal/dialog instead.

## Example

**Collection: Foo**
- Object already exists in collection
- Object is not yet annotated in collection
- You are the owner

**Collection: Bar**
- Object not yet in collection
- You are the owner

**Collection: Baz**
- Object already exists in collection
- Object has 5 annotations in collection (by 3 people)
- Collection owned by Jane Doe

**Comment by Grizzly127** [comment]
### objects/collections/institutions

We thought about filtering between objects, collections and institutions: 
<img width="338" alt="Bildschirmfoto 2024-12-05 um 11 47 38" src="https://github.com/user-attachments/assets/e6b088ba-14eb-4667-9077-ef129889d774">

I thought about reorganizing it on the users/institutional profile more like on sketchfab/pinterest/instagram (other platforms with collections): 
https://github.com/Kompakkt/Repo/issues/43#issuecomment-2413519253

On the explore page we could make searching for users/institutions at the explore page could look like this:
in between the search results here you can easily distinguish if it is a search for an object/collection or a profile. 
the filter option to look for collections and/or objects is kept here. 
<img width="542" alt="Bildschirmfoto 2024-12-05 um 12 36 03" src="https://github.com/user-attachments/assets/f39a0dd3-7fe3-4de4-944e-1144ccbec9da">

@lozanaross @ZetOE @HeyItsBATMAN

**Comment by Grizzly127** [comment]
### access options
<img width="426" alt="Bildschirmfoto 2024-12-05 um 12 43 21" src="https://github.com/user-attachments/assets/6a0fa57f-942f-4b9a-b8a7-d7a11d5beccc">

I'm wondering if it is a use case to look for the different licenses? 
For most users only the information whether the object is downloadable could be interesting and something the user would filter for, right? 

@lozanaross @ZetOE @HeyItsBATMAN

**Comment by Grizzly127** [comment]
@HeyItsBATMAN 
Yes, maybe a dialog makes sense for the annotation options, if we want to display more information. I'll mock up a design suggestion for it.

**Comment by Grizzly127** [comment]
Here are my first ideas: 

- the user is asked where he wants to make the annotation
- We use the info box, we already used for the viewer to give more information about what to do
- the user can create a new collection here or add the object to a collection, where the object is not yet in (then he gets a list about his other existing collections) 
- OR the user chooses one of the collections where the object is already in AND the user has permission (collections where the user is not able to annotate are not shown) 
- it is separated between collections where the user is the owner and collections where the user is not the owner, but has permission to annotate
<img width="342" alt="Bildschirmfoto 2024-12-05 um 14 44 21" src="https://github.com/user-attachments/assets/386f1335-aa48-477d-9f62-94480ffea9ad">
<img width="343" alt="Bildschirmfoto 2024-12-05 um 14 44 26" src="https://github.com/user-attachments/assets/c772447e-2970-43ae-a0e4-3d4788441d3e">

Let me know if this makes sense to you and what can be improved. 
@lozanaross @ZetOE @HeyItsBATMAN
